COPY/usr/bin/manageiq-db-ready

```diff
@@ -25,8 +25,10 @@ end
 def manageiq_db_ready?(db_yaml, env = "production")
   host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
   password = ManageIQ::Password.try_decrypt(password)
+  host_arg = host.present? ? "--host=#{host} " : ""  # If not present, let postgres decide to use localhost/socket
+  port_arg = port.present? ? "--port=#{port} " : ""
 
-  results = `PGPASSWORD=#{password} psql --host=#{host} --port=#{port || 5432} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
+  results = `PGPASSWORD=#{password} psql #{host_arg}#{port_arg} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
```

Comment:
When we end up hitting this the database.yml section looks like:
```
production:
  adapter: postgresql
  encoding: utf8
  username: root
  pool: 5
  wait_timeout: 5
  min_messages: warning
  database: vmdb_production
```

So there is no password either, I'm not sure that this will be able to connect without a password.

Reply:
Right, so that would be an optional env var like the host and port are here.